src/base/static/components/organisms/right-sidebar.js

```diff
@@ -61,21 +70,19 @@ const RightSidebar = props => {
       isRightSidebarVisible={props.isRightSidebarVisible}
     >
       <RightSidebarInnerContainer>
-        <ToggleSidebarButton
+        <Button
+          aria-label="toggle sidebar open"
+          css={toggleSidebarStyles}
```

Comment:
Emotion offers a new `css` prop: https://emotion.sh/docs/css-prop

which is the "new" way of styling components, but I'm hesitant to use it. We'd have to add the `/** @jsx jsx */` pragma at the top of those files, which causes some interferences with Babel and other libs, like Typescript. 

So I think we should stick with the current way of setting styles, using the `styled` library, until the dust settles on the which methodology should be used. I'm leaving the `css` implementation in here as an experiment, but it should be easy to revert back if we really need to get ride of the `css` props.